pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -72,21 +72,8 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, pr *v1beta1.PipelineRun)
 
 	// Get TaskRun names depending on whether embeddedstatus feature is set or not
 	var trs []string
-	if len(pr.Status.ChildReferences) == 0 || len(pr.Status.TaskRuns) > 0 || len(pr.Status.Runs) > 0 { //nolint:all //incompatible with pipelines v0.45
-		for trName, ptrs := range pr.Status.TaskRuns { //nolint:all //incompatible with pipelines v0.45
-			// TaskRuns within a PipelineRun may not have been finalized yet if the PipelineRun timeout
-			// has exceeded. Wait to process the PipelineRun on the next update, see
-			// https://github.com/tektoncd/pipeline/issues/4916
-			if ptrs.Status == nil || ptrs.Status.CompletionTime == nil {
-				logging.FromContext(ctx).Infof("taskrun %s within pipelinerun is not yet finalized: embedded status is not complete", trName)
-				return nil
-			}
-			trs = append(trs, trName)
-		}
-	} else {
-		for _, cr := range pr.Status.ChildReferences {
-			trs = append(trs, cr.Name)
-		}
```

Comment:
Are we zapping this because the option between child references and status.Taskruns only existed in v1beta1?

Reply:
And in v1, it is converted?

Reply:
Yes, exactly - we cannot actually do this anymore if we listen for v1 objects because the v1 objects don't have this field